environments/values.yaml

```diff
@@ -0,0 +1,18 @@
+---
+metrics:
+  enabled: true
+
+serviceAccount:
+  enabled: true
+
+configMaps:
+  enabled: true
+  volumeMounts:
+    enabled: true
+  configs:
+    eventrouter:
+      name: "config.yaml"
+      isFile: true
+      mountPath: /etc/eventrouter
+      data:
+        sink: glog
```

Comment:
Is this a generic values file to be used later for other environments?

Reply:
Can be.  I am going to rename to the appropriate environments as I work on that part of the CI/CD pipeline.